extensions/vscode/webview/src/main.ts

```
@@ -48,6 +56,83 @@ declare global {
   const vscode: WebviewApi | undefined
 }
 
+// Global VSCode API instance
+let vscode_api: WebviewApi | null = null
+let current_app: MatterVizApp | null = null
+
+// Initialize VSCode API once
+const get_vscode_api = (): WebviewApi | null => {
+  if (vscode_api) return vscode_api
+
+  if (typeof acquireVsCodeApi !== `undefined`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)